### ISSUE TITLE
Revert 'Downloaded Only' mode to how it used to function

### ISSUE THREAD

**Issue by MD-1909** [jobobby04/TachiyomiSY#1009]
### Describe your suggested feature

Before SY v1.10 or Tachi v15, the 'Downloaded Only' toggle would show you the mangas with downloaded chapters and within each entry, just the downloaded chapters. However, you could continue reading back or ahead of the downloaded content (it would start loading pages similar to reading online), as long as you had an internet connection.
![Screenshot_20240114_141903_Tachiyomi](https://github.com/jobobby04/TachiyomiSY/assets/46313081/9d5d14f1-6557-4bf2-96e5-da72860aceb5)

But now, using downloaded only does not allow you to backread online chapters and instead forces you to go and turn the toggle off, then go back to the manga, scroll all the way to the chapter you need and then open it.
![Screenshot_20240114_141858_TachiyomiSY](https://github.com/jobobby04/TachiyomiSY/assets/46313081/ccd2479f-6bef-4b40-b91a-8272bc485c22)

This feels like a step backwards and it really ruins my reading experience. I'm sure that not a lot of people care about this but if it isnt too difficult to revert this to how it was before, I would greatly appreciate it.

TL; DR - Downloaded only is working as it should, I just dislike the fact that it revokes your access to any of the other chapters as if they don't exist in the manga at all, which was not a case for 3 whole years.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the extension's repository.
- [X] I have updated the app to version **[1.10.1](https://github.com/jobobby04/tachiyomisy/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by NeKoOuO** [comment]
+1

**Comment by jobobby04** [comment]
I like the download-only filter as it is
